Kernel/WorkQueue.cpp

```diff
@@ -14,11 +14,13 @@ namespace Kernel {
 
 WorkQueue* g_io_work;
 WorkQueue* g_ata_work;
+WorkQueue* g_power_management_work;
 
 UNMAP_AFTER_INIT void WorkQueue::initialize()
 {
     g_io_work = new WorkQueue("IO WorkQueue Task"sv);
     g_ata_work = new WorkQueue("ATA WorkQueue Task"sv);
+    g_power_management_work = new WorkQueue("Power Management WorkQueue Task"sv);
```

Comment:
Hmm. Do we really need a whole new work queue just to service Power state changes? This change isn't mentioned in the commit message, which only talks about restricting access to only those processes that have a jail